### DISCUSSION TITLE
内置算法输出结果可读性

### DISCUSSION THREAD

**Discussion by doudoubobo** [alibaba/GraphScope#483] [Q&A] [answered]
当使用基于属性图的内置算法时（i.e.property_sssp）:输出结果时并不知道context中每张表保存结果的列名是什么，需要到源码中去查。
同时，在源码中描述列名为‘dist_’,后缀的labelid在使用时并不知道。

**Comment by siyuan0322** [comment, marked as answer]
Resolved by #734
